docs/usage/private-npm-modules.md

```diff
@@ -69,14 +69,16 @@ module.exports = {
       // https://www.jfrog.com/confluence/display/JFROG/npm+Registry
       // Will be passed as `//artifactory.my-company.com/artifactory/api/npm/npm:_auth=<TOKEN>` to `.npmrc`
```

Comment:
```suggestion
      // Will be passed as `//artifactory.my-company.com/artifactory/api/npm/npm/:_auth=<TOKEN>` to `.npmrc`
```

The samples there are using trainling slash too. @rarkins maybe we can implicit add the trailing slash too?
![image](https://user-images.githubusercontent.com/1798109/119824017-bb165500-bef5-11eb-8ec8-d8e9e9c5b762.png)

Reply:
But are those examples just for artifactory or apply to every registry?